daemon/mlops-agent-android.c

```diff
@@ -15,6 +15,8 @@
 #include "log.h"
 #include "mlops-agent-interface.h"
 #include "mlops-agent-internal.h"
+#include "../libsqliteX/include/sqlite3.h"
+#include "../libsqliteX/include/sqlite3ext.h"
```

Comment:
sqlite is an external package.
use ```#include <headername.h>``` and add includepath to the script (.mk file). this will help moving sqlite to another repo.